yaml_armhf/queueworker.yml

```diff
@@ -21,4 +21,8 @@ spec:
           value: "30s"
         - name: faas_function_suffix
           value: ".openfaas-fn"
+        - name: basic_auth
+          value: "false"
+        - name: secret_mount_path
+          value: "/etc/openfaas"
```

Comment:
Don't think this is right?

Reply:
should we remove `secret_mount_path`? I am not sure how `secret_mount_path` works for armhf.

Reply:
I'm not sure. @rdimitrov please can you test?

Reply:
Sure, I'll give it a go :+1:

Reply:
@rdimitrov any update? Thanks